queries/cpp/highlights.scm

```diff
@@ -63,6 +63,10 @@
 (operator_name) @function
 "static_assert" @function.builtin
 
+(function_definition
+    declarator: (operator_cast
+      ("operator" @function _)))
+
```

Comment:
shouldn't we just always highlight `"operator"` as `@keyword.operator` with the rest of the query not being needed?

Reply:
Hm. Not sure. Both of these approaches would make the highlighting consistent between all operator uses, but `operator` is technically not a keyword, AFAIK. 

Previously,  all operator overload functions were highlighted correctly as functions / members, as this query would also do this for this special case.

But thinking about it, to be correct this query must be just:

```suggestion
(operator_cast ("operator" @function _))
```

In the end I'm fine with `@keyword`.  `@keyword.operator` might be confusing, because the "real" operator is following `"operator"` directly, e.g. : 

```cpp
T operator->()
//^^^^^^^^^^ All the same color now
```

Reply:
I see... Before all uses of operator were highlighted as "function" anyways. So we could go with your original function or just `"operator" @function`, right?